### DISCUSSION TITLE
Uploading is important

### DISCUSSION THREAD

**Discussion by CEbbinghaus** [photoview/photoview#897] [General]
Pardon the rant but after finding another very promising project that also runs off the file system and doesn't support uploading I have decided to make the mistake of posting my opinion online. I like and even encourage the direction photo view has gone with respect to being a RO view of the image files present on the server. However this design has been killing me recently. Most importantly in the Accessibility aspect which just makes it so much more difficult to deal with.

For context I am wanting to run some services for me and my family. I already use Funkwhale for music but was hoping to get something more convenient & found navidrone. Looks amazing but beyond the missing LDAP support it also doesn't allow uploading of music. This is a problem because the most technical person in our family is my dad and he struggles to operate even semi modern software let alone having to use 2 different pieces of software (one purely for uploading and one purely for viewing). The rest are all worse and thinking about it from an accessibility standpoint the trend has been going far into the direction of "as much on one site as possible". Anybody that has used Facebook knows they don't have to upload at `upload.facebook.com` so they aren't particularly understanding nor receptive of a design that requires more than going to `example.com` and having both the view and the upload in one place.

while from a technical/security/SOC point of view excluding the upload is a good thing. It really decreases the usability once taken outside of the vacuum of "view images online".

TL;DR: Family doesn't like having to use 2 apps.

**Comment by jult** [comment]
So, what is your point? I've read it 3 times, but still don't see what you're trying to say here.

**Reply by CEbbinghaus** [reply]
that these apps should also support uploading of files. Even if its a addon or not recommended or whatever. At this point though I give up though

**Comment by viktorstrate** [comment]
See also #919
